unittests/src/test/java/com/google/appengine/samples/BaseTestConfiguration.java

```diff
@@ -21,5 +21,5 @@
 import org.junit.rules.Timeout;
 
 public abstract class BaseTestConfiguration {
-  @Rule public Timeout testTimeout = new Timeout(5, TimeUnit.MINUTES);
+  @Rule public Timeout testTimeout = new Timeout(10, TimeUnit.MINUTES);
```

Comment:
Increasing this means that each test can take upto 10m, perhaps we should only do this on the test that is failing?